docs/content/preview/migrate/install-yb-voyager.md

```diff
@@ -20,10 +20,21 @@ The following sections describe the prerequisites for installing YugabyteDB Voya
 
 You can install YugabyteDB Voyager on the following:
 
-- CentOS 7
-- Ubuntu 18.04, 20.04
+- RHEL 7/8
+- CentOS 7/8
+- Ubuntu 18.04, 20.04, 22.04
 - MacOS (currently supported only for PostgreSQL source database)
 
+    {{< note title = "note" >}}
+
+You will get the following error if you use MySQL/Oracle as the source database on MacOS:
+
+```output
+"ora2pg" not found. Check if it is installed and included in the path.
+```
+
```

Comment:
Somewhere here suggest the user to use yb-voyager docker container to migrate from Oracle/MySQL on macOS.

Reply:
@amit , do we need to include the note. We can mention for MacOS as follows:

Reply:
```
- macOS (For MySQL/Oracle source databases, [install yb-voyager](#install-yb-voyager) using the Docker option.)
```
should be enough.

Reply:
Thank you ! This is very clear.